### ISSUE TITLE
Make ScheduledSchedules Serialize + Deserialize

### ISSUE THREAD

**Issue by Bendi11** [Bendi11/starfleet#2]
Ensure that the ScheduledSchedule saves/loads its last update date and time with serde. This way, updates scheduled to happen once a day actually happen.

## Note:
This would also mean that upon starting the game after not playing for a while every system would be triggered because they use real time instead of game time played. A better solution would be to have a clock only ticking when the game runs and track schedule update times based on that

**Comment by Bendi11** [comment]
An alternative is to have the universe play 'catch-up' when it first starts, temporarily lowering the duration of scheduled schedules to a much lower value and running them based on how much time has passed. E.x: # Of times to run schedule = duration % time since last update. To ensure that all systems are still run in the relatively right order instead of just running the fast schedules a billion times in a row, change the logic of the scheduled scheduler to instead of just checking if the correct amount of time passed do:

```rust
if (self.last_update - chrono::Utc::now()) > self.time {
    self.last_update += self.time; //Instead of self.last_update = chrono::Utc::now(), this will actually run the scheduled schedule multiple times (though not in a row exactly  it still waits until the run() function is called again, awesome)
    self.schedule.run();
}
```

**Comment by Bendi11** [comment]
Fixed with commit 22812e639860ec68af0595da9de9f3e742350558
